src/components/pages/maintenance/maintenance.scss

```diff
@@ -4,11 +4,64 @@
 @import 'src/styles/themes';
 
 .maintenance-container {
```

Comment:
These styles are mainly place holders until the actual maintenance page work is done.